requirements/tests.in

```diff
@@ -11,3 +11,4 @@ pytest
 ptpython
 requests-mock
 deepdiff
+substrate-interface
```

Comment:
No need to have it here. tests.in include all the dependencies of base.in, thus substrate-interface should be included.

Reply:
I think it was failing for me for some reason, but I will test to see if it's OK without it

Reply:
Done